[2023-01-17T22:40:46.205+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_data_etl_pipeline.extract_integrate_dataset manual__2023-01-17T22:40:38.145844+00:00 [queued]>
[2023-01-17T22:40:46.268+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_data_etl_pipeline.extract_integrate_dataset manual__2023-01-17T22:40:38.145844+00:00 [queued]>
[2023-01-17T22:40:46.269+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-17T22:40:46.270+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2023-01-17T22:40:46.270+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-17T22:40:46.324+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): extract_integrate_dataset> on 2023-01-17 22:40:38.145844+00:00
[2023-01-17T22:40:46.337+0000] {standard_task_runner.py:55} INFO - Started process 9425 to run task
[2023-01-17T22:40:46.344+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'accidents_data_etl_pipeline', 'extract_integrate_dataset', 'manual__2023-01-17T22:40:38.145844+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpisrhq5vo']
[2023-01-17T22:40:46.349+0000] {standard_task_runner.py:83} INFO - Job 44: Subtask extract_integrate_dataset
[2023-01-17T22:40:46.498+0000] {task_command.py:389} INFO - Running <TaskInstance: accidents_data_etl_pipeline.extract_integrate_dataset manual__2023-01-17T22:40:38.145844+00:00 [running]> on host 7fc7752d3cb4
[2023-01-17T22:40:46.651+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=accidents_data_etl_pipeline
AIRFLOW_CTX_TASK_ID=extract_integrate_dataset
AIRFLOW_CTX_EXECUTION_DATE=2023-01-17T22:40:38.145844+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-17T22:40:38.145844+00:00
[2023-01-17T22:40:46.657+0000] {logging_mixin.py:137} INFO - got file paths successfully
[2023-01-17T22:40:52.106+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/***/operators/python.py:192 DtypeWarning: Columns (10) have mixed types.Specify dtype option on import or set low_memory=False.
[2023-01-17T22:40:55.240+0000] {logging_mixin.py:137} INFO - loaded CSVs files to df successfully
[2023-01-17T22:41:28.103+0000] {logging_mixin.py:137} INFO - data integrated successfully
[2023-01-17T22:41:28.186+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-17T22:41:28.493+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=accidents_data_etl_pipeline, task_id=extract_integrate_dataset, execution_date=20230117T224038, start_date=20230117T224046, end_date=20230117T224128
[2023-01-17T22:41:28.699+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-01-17T22:41:28.882+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
